.github/actions/setup_python/action.yml

```diff
@@ -31,8 +31,9 @@ runs:
 
     - if: ${{ runner.os == 'Linux' && runner.arch == 'ARM64' }}
       name: Setup Python ${{ inputs.version }}
-      uses: deadsnakes/action@v3.0.1
+      uses: akashchi/deadsnakes-action@c448323fa5caf174c8031875f5dbec027a4a3ce6
```

Comment:
Sadly, the original deadsnakes action does not work inside a container as it uses `github.action_path` - it is incorrectly set if actions are executed inside containers: https://github.com/actions/runner/pull/1762#issuecomment-1105843659

Reply:
The fix is introduced [here](https://github.com/akashchi/deadsnakes-action/blob/main/action.yml).